app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SwipeToDeleteCallback.kt

```diff
@@ -47,7 +47,7 @@ class SwipeToDeleteCallback(
     ) {
         super.onChildDraw(c, recyclerView, viewHolder, dX, dY, actionState, isCurrentlyActive)
         val icon = AppCompatResources.getDrawable(recyclerView.context, R.drawable.ic_delete)
-        icon?.setTint(recyclerView.context.getColorFromAttr(R.attr.destructive))
+        icon?.setTint(recyclerView.context.getColorFromAttr(R.attr.swipeDeleteIconColor))
```

Comment:
From the hex values we added, I'm not sure why we need to add a new attr for the delete icon? These all seem to be the same as the `destructive` attrs for each theme?

Reply:
The "destructive" color is used in a handful of other spots which I didn't want to create a side effect for:

<img width="956" alt="Screenshot at Mar 04 08-57-29" src="https://user-images.githubusercontent.com/46655/75892193-8270f400-5df6-11ea-8db1-0a2bf0d374bf.png">

Reply:
Right, but it seems like right now `destructive` is the same hex in all 3 themes as `swipeDeleteIconColor`?